diff --git a/rmake.py b/rmake.py
index 89795c01..73cdf27c 100755
--- a/rmake.py
+++ b/rmake.py
@@ -368,10 +368,10 @@ def config_cmd():
             cmake_options.append(f"-DTENSILE_VERSION={args.tensile_version}")
         if args.upgrade_tensile_venv_pip:
             cmake_options.append(f"-DTENSILE_VENV_UPGRADE_PIP=ON")
-        if not args.merge_architectures:
-            cmake_options.append(f"-DTensile_SEPARATE_ARCHITECTURES=ON")
-        if args.tensile_lazy_library_loading:
-            cmake_options.append(f"-DTensile_LAZY_LIBRARY_LOADING=ON")
+        if args.merge_architectures:
+            cmake_options.append(f"-DTensile_SEPARATE_ARCHITECTURES=OFF")
+        if not args.tensile_lazy_library_loading:
+            cmake_options.append(f"-DTensile_LAZY_LIBRARY_LOADING=OFF")
         if args.tensile_msgpack_backend:
             cmake_options.append(f"-DTensile_LIBRARY_FORMAT=msgpack")
         else:
