diff --git a/rmake.py b/rmake.py
index cc7feb9c..068583c5 100755
--- a/rmake.py
+++ b/rmake.py
@@ -434,9 +434,9 @@ def main():
         fatal("Configuration failed. Not continuing.")
 
     # make
-    exe, opts = make_cmd()
-    if run_cmd(exe, opts):
-        fatal("Build failed. Not continuing.")
+    # exe, opts = make_cmd()
+    # if run_cmd(exe, opts):
+    #     fatal("Build failed. Not continuing.")
 
     # Linux install and cleanup not supported from rmake yet
 
